packages/now-build-utils/src/fs/run-user-scripts.ts

```diff
@@ -65,9 +69,10 @@ export function execAsync(
       child.on('close', (code, signal) => {
         if (code !== 0) {
           return reject(
-            new Error(
-              `Program "${command}" exited with non-zero exit code ${code} ${signal}.`
-            )
+            new NowBuildError({
+              code: `NOW_BUILD_UTILS_EXEC_${code}_${signal}`,
+              message: `Program "${command}" exited with non-zero exit code ${code} ${signal}.`,
```

Comment:
@TooTallNate Should I update this error message too? Since it is always printing both code and signal? Any suggestions here?

Reply:
`code || signal`?

Reply:
Ok fixed so these error messages are the same for both exec and spawn